library/plugins/callback/failure_log.py

```diff
@@ -0,0 +1,52 @@
+import yaml
+import os
+
+
+def log_failure(host, result):
+    """
+    Print any failures to stdout nicely formatted as yaml.
+
+    If the environment variable ANSIBLE_FAILURE_LOG is present
+    a log of all failures in the playbook will be persisted to
+    the file path given in ANSIBLE_FAILURE_LOG.
+    """
+    failure = {"{0}".format(host): dict()}
+    failure[host] = result
+
+    print "\n****Ansible Failure******\n"
+    print yaml.safe_dump(failure)
```

Comment:
So, the above will cause *additional* logging to stdout? I'm not sure if that'll be useful - ansible already logs all this data.